library/general/annotationrestriction/constraint.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: RequiredAnnotations
+metadata:
+  name: ingress-alb-ssl-policy
+spec:
+  match:
+    kinds:
+      - apiGroups: ["extensions", "networking.k8s.io"]
+        kinds: ["Ingress"]
+  parameters:
+    annotationKey: alb.ingress.kubernetes.io/ssl-policy
```

Comment:
Do we want to give `parameters` a field called `annotations` which is a list of objects with `key` and `value`... e.g.:

```
  parameters:
    annotations:
       - key: alb.ingress.kubernetes.io/ssl-policy
         value: ELBSecurityPolicy-TLS-1-2-2017-01
```

That way the structure mirrors requiredlabels and people could require more than one annotation on the same constraint if they want.